### PR TITLE
517 Domains + weights & implementation for Email Gen | Password Gen Increased In Randomness | Error Handling 

### PR DESCRIPTION
**Pull request by B1GBOOM420**
This will address issue #20 
- includes 140+ domains with extensions
- updated password generation to add randomness so its harder to detect

I'm happy to be able to contribute to this project!

Thanks for the opportunity

### CODE REVIEW

**Comment by B1GBOOM420** [comment]
- Include over 400+ domains with extentions 👍

**Comment by B1GBOOM420** [comment]
includes updates for the following pull requests 
#6 
#7 
#12 
#18 

It won't let me tag you guys - HUGE shout out to everyone for their contributions!

**Comment by B1GBOOM420** [comment]
I hope this helps in cleaning up the issues and some PR's! @CybrZone 

Thanks again for making this a community project - I haven't coded this much in years lol

File: phishkiller.py [hunk 1]
```diff
@@ -3,34 +3,39 @@
...
     if NumberOrNo == "Number":
         return name + str(random.randint(1, 100)) + domain
     else:
         return name + domain
 
-def generate_random_password():
-    return ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(8))
+
+def generate_random_password():  # Generate password using uppercase, lowercase, numbers and special characters
+    characters = string.ascii_letters + string.digits + "@$!%^&*"
```
[the remark this answers is not in the document]
Comment (B1GBOOM420) [reply phishkiller.py:58]: Thank you for your suggestion - once I'm back at my desk, I will update that.

Thanks again 🙂

**Comment by PanosoikoGr** [comment]
@B1GBOOM420 I have found a great way to change ip with a python lib, I just did a pull request, the code is not tested but the consept is there(I dont have a url I can run the script to test it...)

**Review by SuperProCz — APPROVED** [review]

**Comment by BRICKRAID** [comment]
What did you use to decide the weights of email domain selection choice? In PR #14 I used millions of users published by each.

**Comment by B1GBOOM420** [comment]
I asked chat GPT to estimate based off the same idea of per/million BUT I found in testing that it was aggressively avoiding the odd domains - so I did some manual updates to try and balance it out a tad more

**Comment by CybrZone** [comment]
Thanks @B1GBOOM420 , I merged it to main, there was a slight issue with the email list as @uber.co.uk was missing a comma , :) just for awareness.

**Pull request merged**